dev/builddeps-veloxbe.sh

```diff
@@ -189,7 +189,7 @@ function build_velox {
 function build_gluten_cpp {
   echo "Start to Gluten CPP"
   cd $GLUTEN_DIR/cpp
-  rm -rf build
+  sudo rm -rf build
   mkdir build
   cd build
```

Comment:
do we really need this change? if you need sudo rm, you will need sudo mkdir too, could you double check it?

Reply:
Thanks for reminding, fixed.  But this is only for removing, so this should not influence the build result.